a.rb

```diff
@@ -3,4 +3,14 @@ class Error < Exception
 
 raise Error
 
-puts "Hello world"
+def write(message, io=STDOUT)
+  io.puts(message)
+end
+
+puts "hogehoge"
+
+io = File.open("foo.txt")
```

Comment:
**[querly]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
File.open("foo.txt")	Use block to read/write file (sample.file.open)